app/controllers/email_application_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class EmailApplicationController < StandardStepsController
```

Comment:
Good idea. I think that this naming is confusing because of the word `Email`. Changed the Step name and related controllers to `SendApplication` at f319808